include/alpaka/vec/Traits.hpp

```diff
@@ -14,6 +14,7 @@
 #include <alpaka/extent/Traits.hpp>
 #include <alpaka/offset/Traits.hpp>
 #include <alpaka/idx/Traits.hpp>
+#include <alpaka/meta/IntegerSequence.hpp>
```

Comment:
this should probably read
```suggestion
#include <utility>
```

Reply:
We still use some methods from `IntegerSequence.hpp`